backend/src/main/java/rocks/inspectit/gepard/agentmanager/configuration/validation/ConfigurationRequestHeaderValidator.java

```diff
@@ -0,0 +1,28 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.agentmanager.configuration.validation;
+
+import static rocks.inspectit.gepard.agentmanager.connection.service.ConnectionService.*;
+
+import java.util.Map;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import rocks.inspectit.gepard.agentmanager.exception.MissingHeaderException;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class ConfigurationRequestHeaderValidator {
+  public static void validateConfigurationRequestHeaders(Map<String, String> headers) {
```

Comment:
Can you add the attributes here as well?

Reply:
Are the x-gepard-attributes mandatory? If not, we should not check for them here.

Reply:
True, they are not mandatory